### PR TITLE
fix for web.whatsapp.com

### PR DESCRIPTION
**Pull request by Myshor**
Added alernative data-asset-intro-image-light.
Changed class for QR code.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4082,10 +4082,10 @@ CSS
 ._3Ye_R {
     color: gray !important
 }
-[data-asset-intro-image] {
+[data-asset-intro-image], [data-asset-intro-image-light] {
     background-image: url(/img/intro-connection_c98cc75f2aa905314d74375a975d2cf2.jpg) !important;
 }
-._11ozL {
+._1QMFu {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4088]: Changing every month? Let's see if we can use a `>` strict CSS3 to target this CSS to make it more reliabler in the next updates for whatsapp. Related what you did with google 9 dots in the right above corner?

Comment (Myshor) [reply src/config/dynamic-theme-fixes.config:4088]: Saw inside class nice canvas with aria-label="Scan me!" but making border for it doesn't work and CSS still doesn't work for parents. 😉 but who knows... maybe I'll find some solution in the future.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4088]: Let's not merge it yet I will have a look at it today to get a more global solution to this.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4088]: `html[dir] .landing-main > :first-child > :nth-child(2) > :first-child`

**Comment by Myshor** [comment]
We will see if it won't change. 😏

**Comment by Gusted** [comment]
Than whatsapp will have a angry mail about normalized CSS 😠

**Pull request merged**